src/content/effective-dart/design.md

```diff
@@ -519,10 +519,24 @@ class in a single library.
 
 ## Classes and mixins
 
-Dart is a "pure" object-oriented language in that all objects are instances of
-classes. But Dart does not require all code to be defined inside a
-class—you can define top-level variables, constants, and functions like
-you can in a procedural or functional language.
+**Note:** As of Dart 3.0, class modifiers (e.g., `final`, `base`, `interface`) 
```

Comment:
We should only include very basic top-level information under this high-level heading. Let's revert this to what was already there, but I would advise adding one bullet with a sentence or two that lets people know that classes can be restricted.

 I believe you've already captured these other details in subsections. If something is here but not in one of those sections, let's move the content to that section.